yarn-project/noir-contracts/src/contracts/test_contract/src/main.nr

```diff
@@ -64,6 +65,37 @@ contract Test {
         context.finish()
     }
 
+    // Test codegen for noir interfaces
+    fn testCodeGen(
+        inputs: PrivateContextInputs,
```

Comment:
Perhaps we should add a similar test for an `open` (public) and an `unconstrained` function?

Reply:
Added for `open`, not for `unconstrained` as discussed on Slack